src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -30,6 +33,10 @@ public void end(boolean interrupted) {
 		intakeSubsystem.intakeReject();
 		intakeSubsystem.indexStop();
 		intakeSubsystem.feederStop();
+
+		if (controls != null && interrupted == false) {
```

Comment:
(Nit: you can use `!interrupted` instead of directly comparing against `false`.)